lib/Dialect/TritonGPU/Transforms/AccelerateAMDMatmul.cpp

```diff
@@ -52,26 +52,51 @@ int getMfmaVersion(MatrixCoreVersion matrixCoreVer) {
   return 0;
 }
 
-SmallVector<unsigned, 2>
-warpsPerTile(tt::DotOp dotOp, const ArrayRef<int64_t> shape, int numWarps,
-             SmallVector<int64_t, 2> shapePerWarp) {
-  // TODO: needs to be updated with appropriate shapePerWarp etc.
+static bool isChainDot(tt::DotOp &dotOp) {
```

Comment:
Maybe change name to something more descriptive, for example `isTransposeChainDotPattern`?
```suggestion
static bool isTransposeChainDotPattern(tt::DotOp &dotOp) {
```

+cc @oplavsic

Reply:
Thanks @alefimov-amd. Changed accordingly.